presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetHiveRecordCursor.java

```diff
@@ -321,10 +320,8 @@ public void close()
             boolean predicatePushdownEnabled,
             TupleDomain<HiveColumnHandle> effectivePredicate)
     {
-        ParquetDataSource dataSource = null;
-        try {
-            FileSystem fileSystem = hdfsEnvironment.getFileSystem(sessionUser, path, configuration);
```

Comment:
This is incorrect. Hadoop file systems cannot be closed since they are singletons.

Reply:
code the same - hdfs file system just parameter of method buildHdfsParquetDataSource